CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to
   - [#3390](https://github.com/bpftrace/bpftrace/pull/3390/)
 - Add `has_key` function for maps
   - [#3358](https://github.com/bpftrace/bpftrace/pull/3358)
+- Add `symbol_source` config to source uprobe locations from either DWARF or the Symbol Table
```

Comment:
It would be fine, but this settings is also going to apply to kprobes in #3301. Do you have a suggestion fitting for both?

Reply:
Oh, ok. Didn't know kprobes were going to be using it too. This is fine then.

Reply:
I'm not so bothered about the name but just want to clarify what the plans are for these config options.

Inlined kprobes are already going to be controlled by the probe_inline config option, right? For non-inlined kprobes, I'm not sure there's any benefit to using LLDB's locations. The reason for uprobes was that the kernel doesn't provide correct stack traces otherwise, but that's not the case for kprobes.

Reply:
To second what @ajor said, would it make sense to extend the `probe_inline` option to choose whether to use DWARF or symbol table for placing the probes? And use symbol table for non-inlined uprobes?

Reply:
There is still a reason to have two separate config options - we want to control these two things independently:
- Uprobes/kprobes tracing inlined functions
- Uprobes using debug info to pick attach points for non-inlined functions (to fix ustacks: #646)

The second one is needed because we've come across a case where BOLT (or similar) messes up the DWARF info for non-inlined functions, so we need a way to fall back to the symbol table in that case.

We shouldn't be using DWARF info for non-inlined kprobes anyway, so this config option only needs to apply to uprobes.

Reply:
Ah, ok, thanks for the clarification.